providers/UserProvider.tsx

```diff
@@ -12,12 +12,14 @@ import {
 import { useAccount } from "wagmi"
 import { getSimilarProfiles, getUserInfo } from "../lib/userInfo"
 import { useRouter } from "next/router"
+import getMetadata from "../lib/getMetadata"
 
 interface userProps {
   getUserData: (address?: string) => Promise<void>
   getUserSimilarProfiles: (address?: string) => Promise<void>
   userInfo: any
   similarProfiles: any
+  metaData: any
```

Comment:
*nit* kinda defeats the purpose of types if we use any, either set appropriate type or just don't set types.

Reply:
```
interface attribute {
value?:string,
trait_type?:string
}
interface metadata {
attributes?:attribute[],
description?:string,
image?:string,
name:?string
}
```

Reply:
fixed 👍